indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -40,8 +42,19 @@ function TimetableSessionsComponent({
     onChange(newValue);
   }
 
+  const minMsg = minimum > 0 ? `at least ${minimum}` : ``;
+  const maxMsg = maximum > 0 ? `no more than ${maximum}` : ``;
+  const and = minimum > 0 && maximum > 0 ? ` and ` : ``;
+  const validatorMsg =
+    maximum > 0 || minimum > 0 ? `Please select ${maxMsg} ${and} ${minMsg} session blocks. ` : ``;
+  const currentSelectionMsg = `You selected ${value.length} session blocks.`;
+
   return (
     <Form.Group styleName="timetablesessions-field">
+      <strong>
+        {validatorMsg}
```

Comment:
I have added a visualization of the minimum and maximum session blocks configured after realising the user has no feedback of the validator before sending the form